udata/core/post/api.py

```diff
@@ -70,11 +78,13 @@ class PostsAPI(API):
     def get(self):
         """List all posts"""
         args = parser.parse_args()
-        return (
-            Post.objects.published()
-            .order_by(args["sort"])
-            .paginate(args["page"], args["page_size"])
-        )
+
+        posts = Post.objects()
+
+        if not (AdminPermission().can() and args["with_drafts"]):
+            posts = posts.published()
```

Comment:
I think we could have this behavior by default, not needing a `with_drafts` parameter?
Since we're mostly going for a logic of getting objects to which you have access by default